### DISCUSSION TITLE
Addition of rivers?

### DISCUSSION THREAD

**Discussion by helenmacdonald** [COSIMA/mom6-regional-scripts#195] [Q&A] [answered]
In the youtube demonstration I saw that there exists code for adding in rivers to the regional MOM. Are there plans to include this in the main branch? Can you please point me to a copy of this code? I am happy to help develop code if it is not quite in a usable format.
Thanks

**Comment by ashjbarnes** [comment, marked as answer]
Hi Helen! Yes we haven't gotten around to this one yet. Here's the link to the old function that does runoff:

https://github.com/ashjbarnes/mom6-regional-scripts/blob/73cce595deab95696e86fbdf1bdbcdfee045cbd1/mom6-regional/regional_model_scripts.py#L377

By the looks of things it shouldn't be too hard to put into the package. @manishvenu and others at NCAR are also looking at adding rivers to the package on their [fork](https://github.com/CROCODILE-CESM/regional-mom6) so may have code of their own too.

**Reply by helenmacdonald** [reply]
Perfect! Thanks for that – just what I was looking for.
